src/redux/categories/categories.js

```diff
@@ -0,0 +1,14 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
The value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux)

    const CHECK_STATUS = 'bookstore/sr/redux/categories/CHECK_STATUS'